base/model/src/main/java/org/eclipse/ditto/base/model/common/ILikeHelper.java

```diff
@@ -0,0 +1,78 @@
+
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.base.model.common;
+
+import java.util.regex.Pattern;
+
+/**
+ * A helper to create "like" patterns.
+ *
+ * @since 2.3.0
+ */
+public final class ILikeHelper {
```

Comment:
Why did you add this class?
This is a complete copy of `LikeHelper` as far as I can see - so this can be reused in your `ILikePredicateImpl`.

Please remove this class again.